english/experience/black-point/stargaze.md

```diff
@@ -0,0 +1,11 @@
+When you concentrate on the black point for some more time you see the point transform.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.